src/json.ts

```diff
@@ -24,8 +24,8 @@ export const jsonRuntype = internalRuntype<unknown>((v, failOrThrow) => {
 /**
  * A String that is valid json
  */
-export function json<T>(rt: Runtype<any>): Runtype<T> {
-  return internalRuntype<T>((v, failOrThrow) => {
+export function json<T>(rt: Runtype<T>): Runtype<T> {
+  return internalRuntype<any>((v, failOrThrow) => {
```

Comment:
This is the change made to the `json` runtype.

We want `json` to return a runtype that is the runtype passed to `json`, after `json` has parsed the given string into JSON. This change allows for that: the type of the param of `json` is the same as its return type, and the `any` generic param given to `internalRuntype` means that `internalRuntype` returns a `Runtype<any>` which can be assigned to `Runtype<T>`.

I'm still somewhat of a TS beginner, and I'm not sure of all the consequences of this change, so please let me know if this is incorrect 🙂